app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx

```diff
@@ -68,7 +70,16 @@ const withBackgroundEncryptedUpload = (Component) => ({ onChange, ...props }) =>
               headers: { 'Content-Type': 'application/octet-stream' },
             }),
           )
-          .then(() => url);
+          .then((response) => {
+            const traceId = response.headers.get('X-Amzn-Trace-Id');
+            addPageAction('documentCapture.asyncUpload', { success: response.ok, traceId });
```

Comment:
should we snake case the trace_id for consistency when we send to newrelic? or do you think it doesn't matter?

```suggestion
            addPageAction('documentCapture.asyncUpload', { success: response.ok, trace_id: traceId });
```

Reply:
Also, the Ruby code calls it `amzn_trace_id` for server requests, as another option for consistency

Reply:
Well, but then I called it plain `trace_id` in my lambda logging code so what I'm saying is, we have no rules for this lol

Reply:
To your original question, it probably doesn't matter in that these are specific to the browser SDK, and we might be okay to allow for some context-specific naming conventions (following JavaScript-typical camelCasing here).

But for our own sake, it could be good to have some consistency in the naming when we're talking about the same thing.

Do you have a preference for one or the other? In the absence of a preference, I might lean toward `trace_id` for reasons of...

1. Shorter
2. A general distaste for abbreviations (ignoring the fact that "Id" is an abbreviation 😅 )

Reply:
I've talked myself in circles on this one. I landed on `trace_id` for the lambdas since it's not vendor-specific, and I do like that it's short as well

Reply:
Renamed in 50a2a7278.